functions/gravityForms/getGfFieldId.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Create a field ID based on GravityForms provided ID.
  *
- * @param   {integer} fieldId number.
+ * @param {integer}  fieldId number.
```

Comment:
I _think_ it's just supposed to be `{number}`? And can we split the `fieldId number` into a separate var name and capitalized comment?

Reply:
I checked and that does appear to be the case. There are some examples shown here that display the `number` used in multiple ways. That does somewhat fall inline with `type of` atleast. 

> And can we split the fieldId number into a separate var name and capitalized comment?

I'm not sure I follow what you're asking. Do you mean rewrite it as `* @param {integer}  fieldId Number.`

Reply:
Basically, just that the var name and comment seem to slide together, so I'd suggest something like `@param {number} fieldId GravityForms field ID.`

Reply:
Awesome, that is what I have in my latest update. I have a quick hot tweaks PR coming your way momentarily.